cmd/boulder-ra/main.go

```diff
@@ -192,6 +201,7 @@ func main() {
 	rai.VA = vac
 	rai.CA = cac
 	rai.SA = sac
+	rai.Publisher = pubc
```

Comment:
We've been trying to deprecate the practice of direct assignment to exported fields. Please make this a constructor argument instead.